admin/tests/integration/components/user-detail-personal-information-test.js

```diff
@@ -1,105 +1,258 @@
+      await render(hbs`<UserDetailPersonalInformation @user={{this.user}}/>`);
 
-    assert.dom('.user__pix-certif-terms-of-service-accepted').hasText('NON');
-  });
+      assert.dom('.user__pix-orga-terms-of-service-accepted').hasText('NON');
+    });
+
+    test('should display "OUI" when user accepted Pix Certif terms of service in read only mode', async function(assert) {
+      this.set('user', { pixCertifTermsOfServiceAccepted: true });
+
+      await render(hbs`<UserDetailPersonalInformation @user={{this.user}}/>`);
+
+      assert.dom('.user__pix-certif-terms-of-service-accepted').hasText('OUI');
+    });
+
+    test('should display "NON" when user not accepted Pix Certif terms of service in read only mode', async function(assert) {
+      this.set('user', { pixCertifTermsOfServiceAccepted: false });
+
+      await render(hbs`<UserDetailPersonalInformation @user={{this.user}}/>`);
+
+      assert.dom('.user__pix-certif-terms-of-service-accepted').hasText('NON');
+    });
+
+    test('should display that user is authenticated from GAR in read only mode', async function(assert) {
+      this.set('user', { isAuthenticatedFromGAR: true });
+
+      await render(hbs`<UserDetailPersonalInformation @user={{this.user}}/>`);
+
+      assert.dom('.user__is-authenticated-from-gar').hasText('NON');
+    });
+
+    test('should display that user is not authenticated from GAR in read only mode', async function(assert) {
+      this.set('user', { isAuthenticatedFromGAR: false });
 
-  test('should display that user is authenticated from GAR', async function(assert) {
-    this.set('user', { isAuthenticatedFromGAR: true });
+      await render(hbs`<UserDetailPersonalInformation @user={{this.user}}/>`);
 
-    await render(hbs`<UserDetailPersonalInformation @user={{this.user}}/>`);
+      assert.dom('.user__is-authenticated-from-gar').hasText('NON');
+    });
 
-    assert.dom('.user__is-authenticated-from-gar').hasText('NON');
+    test('should display the edit and cancel buttons when click on update button', async function(assert) {
+
+      this.set('user', { email: 'john.snow@winterfell.got' });
+
+      await render(hbs`<UserDetailPersonalInformation @user={{this.user}}/>`);
+
+      await click('button[aria-label=\'Modifier\'');
+
+      assert.dom('button[aria-label=\'Editer\'').exists();
+      assert.dom('button[aria-label=\'Annuler\'').exists();
+    });
   });
 
-  test('should display that user is not authenticated from GAR', async function(assert) {
-    this.set('user', { isAuthenticatedFromGAR: false });
+  module('should display the form in editable mode when admin click on update user details', async function() {
+
+    module('should display only fields allowed for update', async function() {
+
+      let user = null;
+
+      hooks.beforeEach(function() {
+        user = EmberObject.create({ lastName: 'harry', firstName: 'John', email: 'john.harry@gmail.com' });
```

Comment:
Pourquoi le prénom est en majuscule et pas le nom de famille ? 
Je ne comprends pas l'intention
```suggestion
        user = EmberObject.create({ lastName: 'Harry', firstName: 'John', email: 'john.harry@gmail.com' });
```